server/rucio.conf.j2

```diff
@@ -52,6 +52,9 @@ CacheRoot /tmp
 {% else %}
  SSLCACertificateFile /etc/grid-security/ca.pem
 {% endif %}
+ SSLCACertificatePath /etc/grid-security/certificates
+ SSLCARevocationPath /etc/grid-security/certificates
+ SSLCARevocationCheck chain
```

Comment:
Please put this line together with the others, i.e., below line 51 since it's necessary to have "SSLCARevocationPath" set, and make it configurable, e.g.,

`SSLCARevocationCheck {{ RUCIO_CA_REVOCATION_CHECK | default('chain') }}`

I know 'chain' is the recommended value but there might cases where people want to set it to something else.

Reply:
Thomas, you're right about paths variables, I missed them. And, of course make check configurable is more flexible. I applied the change